### ISSUE TITLE
multiple GH units

### ISSUE THREAD

**Issue by thecivvie** [erik/lastcast#29]
Is it possible to add support for more than one GH unit on the same network. I have one upstairs that I listen to at nighttime

**Comment by erik** [comment]
Hi, thanks for the feature request!

I've actually had a couple people ask about support for this, and the answer has historically been "just run multiple lastcast instances", but I can do better than that.

~I'll start thinking about how to add support for multiple chromecasts, but I'm not sure when I'll have time to implement this.~

**Comment by erik** [comment]
Implemented this in #30. Going to keep testing this for a few days to make sure there aren't any corner cases I missed before merging.

**Comment by erik** [comment]
Just pushed this feature!

Version `0.5.0` should let you scrobble from multiple devices (at the same time, if you want).

`pip install --upgrade lastcast`

Here's how to update the config:

```diff
  [chromecast]
- name = "cast 1"
+ devices = ["cast 1", "cast 2"]
```
